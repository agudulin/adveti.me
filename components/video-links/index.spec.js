import React from 'react'
import renderer from 'react-test-renderer'

import VideoLinks from './'

const basicProps = {
  videos: [{
    url: 'video-url-1',
    name: 'video-name-1'
  }, {
    url: 'video-url-2',
    name: 'video-name-2'
  }]
}

describe('VideoLinks', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <VideoLinks {...basicProps} />
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
