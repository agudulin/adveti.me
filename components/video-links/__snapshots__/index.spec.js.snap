exports[`VideoLinks renders correctly with lots of video links 1`] = `
<div
  className="video-links"
  data-jsx={1141445683}>
  <a
    className="video-links__item"
    data-jsx={1141445683}
    href="video-url-1">
    video-name-1
    <span
      className="video-links__comma"
      data-jsx={1141445683}>
      ,
    </span>
  </a>
  <a
    className="video-links__item"
    data-jsx={1141445683}
    href="video-url-2">
    video-name-2
  </a>
</div>
`;

exports[`VideoLinks renders correctly with one video link 1`] = `
<div
  className="video-links"
  data-jsx={1141445683}>
  <a
    className="video-links__item"
    data-jsx={1141445683}
    href="video-url-1">
    video-name-1
  </a>
</div>
`;
